hack/testing/check-logs.sh

```diff
@@ -30,18 +30,44 @@ if [ -z "${USE_JOURNAL:-}" ] ; then
     fi
 fi
 
-USE_JOURNAL="${USE_JOURNAL}"    \
-OAL_ELASTICSEARCH_COMPONENT="es" \
-OAL_KIBANA_COMPONENT="kibana"   \
-OAL_ELASTICSEARCH_SERVICE="logging-es" \
-"${OS_O_A_L_DIR}/test/cluster/functionality.sh"
+USE_OPERATOR=${USE_OPERATOR:-false}
+if oc get clusterlogging > /dev/null 2>&1 ; then
+	USE_OPERATOR=${USE_OPERATOR:-true}
+fi
 
-if [ "$1" = "true" ]; then
-  # There is an ops cluster set up, so we
-  # need to verify it's functionality as well.
-  USE_JOURNAL="${USE_JOURNAL}"        \
-  OAL_ELASTICSEARCH_COMPONENT="es-ops" \
-  OAL_KIBANA_COMPONENT="kibana-ops"   \
-  OAL_ELASTICSEARCH_SERVICE="logging-es-ops" \
-  "${OS_O_A_L_DIR}/test/cluster/functionality.sh"
+if [ "$USE_OPERATOR" = true ] ; then
+    if [ "$1" = "true" ]; then
+        USE_JOURNAL="${USE_JOURNAL}"    \
+        OAL_ELASTICSEARCH_COMPONENT="es" \
+        OAL_KIBANA_COMPONENT="kibana-app"   \
+        OAL_ELASTICSEARCH_SERVICE="elasticsearch-app" \
+        "${OS_O_A_L_DIR}/test/cluster/functionality.sh"
+        USE_JOURNAL="${USE_JOURNAL}"    \
+        OAL_ELASTICSEARCH_COMPONENT="es-ops" \
```

Comment:
for the operator this will be the same as the cluster name `elasticsearch` `elasticsearch-app` `elasticsearch-infra`
https://github.com/openshift/elasticsearch-operator/blob/master/pkg/k8shandler/desirednodestate.go#L175

Reply:
Since the operator will not split the installation we can likely remove this completely

Reply:
Are we supporting an app/ops deployment at all anymore?